specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json

```diff
@@ -4909,6 +4951,13 @@
               "$ref": "#/definitions/ExtendedResourceProperties"
             }
           ]
+        },
+        "options": {
```

Comment:
>options [](start = 9, length = 7)

So these are only returned on GETs?

Reply:
When creating a resource, the swagger already allows to specify "options" (through PUT). This change is adding these options to the GET response so that an alias is created for policy purposes.